spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -453,9 +453,7 @@ In the case of `@StreamListener`, the `MessageConverter` mechanism will use the
 As with other Spring Messaging methods, method arguments can be annotated with `@Payload`, `@Headers` and `@Header`.
 
 [NOTE]
-====
 For methods which return data, you must use the `@SendTo` annotation to specify the output binding destination for data returned by the method:
-====
```

Comment:
Why did you remove these? `NOTE`s have 2 syntaxes

Admonition paragraph
```
NOTE:foobar
```
and 

Admonition Block
```
[NOTE]
====
foobar
====
```

See http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#admon-bl